### ISSUE TITLE
Bridge: Do not automatically show the API Token in the menu

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#2331]
Right now this is what you see when you click on the menu on the top right side:

![image](https://user-images.githubusercontent.com/56065213/93072023-229ab600-f681-11ea-99cb-318670b03cb2.png)

While I do see the benefit of having it, it's certainly not good to have it visible by default. For instance, if you record a video or if you have a Zoom Session active where you present this feature, and you click that menu, you have automatically shown the world your API token.

To prevent this from happening, we could create a button called "Reveal token", and only after pressing this button, the token is visible for a specified amount of time (e.g., 60 seconds), and then gets hidden again.

# Definition of Done

- [ ] Introduce a button called "Reveal token" instead of showing the token
- [ ] Once the button is pressed, show the token and the copy button as usually
- [ ] After a pre-defined time (e.g., 60 seconds), it should hide the token again

**Comment by christian-kreuzberger-dtx** [comment]
I just found out that this is already covered as a sub-task in #2257, therefore I'm closing my issue.
